frontend/src/modules/search/components/SearchBox.js

```diff
@@ -78,6 +88,42 @@ export class SearchBoxBase extends React.Component<InternalProps, State> {
         document.removeEventListener('keydown', this.handleShortcuts);
     }
 
+    getInitialStatuses() {
```

Comment:
If flow doesn't complain, that's because it's able to deduce that from the code. In which case, no point in making it explicit, I think. At least that's what the folks from AMO recommended: don't write flow types that are not needed.